CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   to allow using the `restricted` and `restricted-v2` SCCs.
 - The service account "central", which is used by the central deployment, will now include `get` and `list` access to the following resources in the namespace where central is deployed to:
   `pods`, `events`, and `namespaces`. This fixes an issue when generating diagnostic bundles to now correctly include all relevant information within the namespace of central.
+- ROX-13265: Fix missing rationale and remediation texts for default policy "Deployments should have at least one ingress Network Policy" on new installations
```

Comment:
The changelog indicates `on new installations` but there is a migration being added through this PR. Migrations are only needed to make changes to existing installations.

Reply:
I would just remove "on new installations". This fix will add it for new **and** any older installations that missed it last time.

Reply:
Done